### PR TITLE
ARO-1476: Default to OVNKubernetes

### PR DESCRIPTION
**Pull request by lranjbar**
[ARO-1476](https://issues.redhat.com//browse/ARO-1476)

To verify this PR we need to install new ARO cluster and verify NetworkType is OVNKubernetes.

### CODE REVIEW

File: pkg/installer/generateconfig.go [hunk 1]
```diff
@@ -110,7 +110,12 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 		masterZones = []string{""}
 	}
 
-	SoftwareDefinedNetwork := string(api.SoftwareDefinedNetworkOpenShiftSDN)
+	// Set NetworkType to OVNKubernetes by default
+	SoftwareDefinedNetwork := string(api.SoftwareDefinedNetworkOVNKubernetes)
```
Comment (facchettos) [inline pkg/installer/generateconfig.go:114]: This should be `softwareDefinedNetwork` , capitalized names are for exported variables/functions

Comment (bennerv) [reply pkg/installer/generateconfig.go:114]: @facchettos can you put in a PR for it?  I'll cherry-pick it onto the release branches.

**Pull request merged**